tests/cross_functional/krkn_chaos/test_krkn_hog_scenarios.py

```
@@ -238,19 +212,14 @@ def test_krkn_hog_multi_stress_chaos(
             )
             raise
         except Exception as e:
-            log.error(f"❌ Multi-stress hog chaos failed for {node_type}: {e}")
+            logger.exception(f"Multi-stress hog chaos failed for {node_type}: {e}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)